### PR TITLE
Copy to catalogue item #178

### LINKED ISSUE #178: Copy to catalogue item

### PR DESCRIPTION
**Pull request by joshuadkitenge**
## Description

A user should be able to copy a catalogue item to another catalogue category

## Testing instructions

Add a set up instructions describing how the reviewer should test the code

- [ ] Review code
- [ ] Check Actions build
- [ ] Review changes to test coverage
- [ ] copy a catalogue to a new directory and the same 
- [ ] error states 
- [ ] {more steps here}

## Agile board tracking

closes #178

### CODE REVIEW

File: src/catalogue/items/catalogueItemDirectoryDialog.component.test.tsx [hunk 1]
```diff
@@ -220,11 +221,163 @@ describe('catalogue item directory Dialog', () => {
...
+        cost_to_rework_gbp: null,
+        days_to_replace: 7,
+        days_to_rework: null,
+        description: 'Precision energy meters for accurate measurements. 26',
+        drawing_link: null,
+        drawing_number: null,
+        id: '89',
+        is_obsolete: true,
+        item_model_number: null,
+        manufacturer: {
```
Comment (MatteoGuarnaccia5) [inline src/catalogue/items/catalogueItemDirectoryDialog.component.test.tsx:255]: should be `manufactuer_id` now

File: src/catalogue/items/catalogueItemDirectoryDialog.component.test.tsx [hunk 2]
```diff
@@ -220,11 +221,163 @@ describe('catalogue item directory Dialog', () => {
...
+        cost_to_rework_gbp: 89,
+        days_to_replace: 7,
+        days_to_rework: 60,
+        description: 'Precision energy meters for accurate measurements. 27',
+        drawing_link: null,
+        drawing_number: null,
+        id: '6',
+        is_obsolete: false,
+        item_model_number: null,
+        manufacturer: {
```
Comment (MatteoGuarnaccia5) [inline src/catalogue/items/catalogueItemDirectoryDialog.component.test.tsx:280]: same here

File: src/catalogue/items/catalogueItemDirectoryDialog.component.test.tsx [hunk 3]
```diff
@@ -220,11 +221,163 @@ describe('catalogue item directory Dialog', () => {
...
+        cost_to_rework_gbp: null,
+        days_to_replace: 7,
+        days_to_rework: null,
+        description: 'Precision energy meters for accurate measurements. 26',
+        drawing_link: null,
+        drawing_number: null,
+        id: '89',
+        is_obsolete: true,
+        item_model_number: null,
+        manufacturer: {
```
Comment (MatteoGuarnaccia5) [inline src/catalogue/items/catalogueItemDirectoryDialog.component.test.tsx:319]: same here

File: src/catalogue/items/catalogueItemDirectoryDialog.component.test.tsx [hunk 4]
```diff
@@ -220,11 +221,163 @@ describe('catalogue item directory Dialog', () => {
...
+        cost_to_rework_gbp: 89,
+        days_to_replace: 7,
+        days_to_rework: 60,
+        description: 'Precision energy meters for accurate measurements. 27',
+        drawing_link: null,
+        drawing_number: null,
+        id: '6',
+        is_obsolete: false,
+        item_model_number: null,
+        manufacturer: {
```
Comment (MatteoGuarnaccia5) [inline src/catalogue/items/catalogueItemDirectoryDialog.component.test.tsx:344]: same here

**Review by MatteoGuarnaccia5 — CHANGES_REQUESTED** [review]
Same thing as in #173 with the manufacturer in tests that should be`manufacturer_id`

**Review by joelvdavies — CHANGES_REQUESTED** [review]
Two minor comments, Apart from the query invalidation works well.

**Review by MatteoGuarnaccia5 — APPROVED** [review]
LGTM

**Review by joelvdavies — APPROVED** [review]

**Pull request merged**